### DISCUSSION TITLE
Keybindings when renaming

### DISCUSSION THREAD

**Discussion by rafpaf** [neoclide/coc.nvim#4969] [Q&A]
It would be nice if I could use keybindings inside the variable-renaming field that pops up using `coc-rename`. Maybe a new buffer could open, just for editing the variable name. Is this currently possible with this or any alternative plugin?

**Comment by fannheyward** [comment]
> new buffer could open

No such feature. You can set `coc.preferences.promptInput` to `false`, this will use the command line are to input.
